include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -14708,6 +14717,13 @@
           "extensions" : [ "SPV_INTEL_kernel_attributes" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "FPGAKernelAttributesv2INTEL",
+          "value" : 6161,
+          "capabilities" : [ "FPGAKernelAttributesINTEL" ],
+          "extensions" : [ "SPV_INTEL_kernel_attributes" ],
+          "version" : "None"
+        },
```

Comment:
Did the headers generate correctly after this change?  I see the new RegisterMapInterfaceINTEL enum, but not this new FPGAKernelAttributesv2INTEL capability.

Reply:
How and where do I check that? Do I have to run something locally? @MrSidims do you know how I can check the generated headers? I am going to try the steps shown in the repo README file.

Reply:
Could be because I did not add `FPGAKernelAttributesv2INTEL` to spirv.hpp11. Adding now.

Reply:
Thanks for catching this, I realize now that `FPGAKernelAttributesv2INTEL` should have been added everywhere just like `FPGAKernelAttributesINTEL`. Updated all files.

Reply:
Hmm, did you regenerate the headers after making this change?  I'm seeing an error when I regenerate headers:

> Error: "Capability" enumerant "FPGAMemoryAccessesINTEL" is out of order. It has value 5898 but follows the enumerant with value 6161

Instructions to regenerate the headers can be found [here](https://github.com/KhronosGroup/SPIRV-Headers#generating-headers-from-the-json-grammar-for-the-spir-v-core-instruction-set).

Reply:
No I did not. Looking into it now.

Reply:
Looks like there is a positioning requirement. Updated, and now the header generation command does not error out.

Reply:
Thanks, I don't see the out-of-order error anymore, but it looks like the `FPGAKernelAttributesv2INTEL` capability is now in the wrong place - see below.

Reply:
Updated in all files.